CHANGELOG.md

```diff
@@ -160,5 +160,5 @@ Read more about it in the [docs](https://hasura.io/docs/1.0/graphql/manual/auth/
 
 - server: check expression in update permissions (close #384) (rfc #3750) (#3804)
 - console: show pre-release update notifications with opt out option (#3888)
-- Allow special characters in JSON path query argument with bracket `[]` notation, e.g `obj['Hello World!']` (#3890)
+- Allow special characters in JSON path query argument with bracket `[]` notation, e.g `obj['Hello World!']`. Single `$` can be accepted as root object (#3890)
```

Comment:
1. Why is this added to `v1.2.0-beta.1` ?

2. I don't think we need a changelog which fixes a bug after v1.2.0-beta.4. Even if it were required, it should go with the changelog entry for: https://github.com/hasura/graphql-engine/pull/3892